conversion/now.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Also, now.json should be moved to the root later on once we have the structure set.